### PR TITLE
refactoring of visits

### PR DESCRIPTION
**Pull request by valeriyapetrova**

### CODE REVIEW

**Review by Rub1st — COMMENTED** [review]
Please write spec on another actions of `VisitsController` and merge conflicts

File: spec/get_visits_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'get all visits route', type: :request do
+  let!(:visits) { FactoryBot.create_list(:random_visit, 20) }
+
+  before { get '/api/v1/visits' }
+
+  it 'returns all visits' do
```
Comment (TehanovAnton) [inline spec/get_visits_spec.rb:7]: As we decided, we remove controller tests at this moment. We will back to it later

File: spec/get_visits_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'get all visits route', type: :request do
+  let!(:visits) { FactoryBot.create_list(:random_visit, 20) }
+
+  before { get '/api/v1/visits' }
```
Comment (TehanovAnton) [inline spec/get_visits_spec.rb:5]: Lets use generators like 'rails g rspec:model <model_name>' for building spec structure

**Comment by TehanovAnton** [comment]
I propose to assign "assignees" in PRs and leave a little description

**Review by GEK62 — APPROVED** [review]

**Pull request merged**